internal/controller/rhdh/backstage.go

```diff
@@ -36,121 +34,166 @@ var ConfigMapNameAndConfigDataKey = map[string]string{
 	AppConfigRHDHDynamicPluginName: "dynamic-plugins.yaml",
 }
 
-func CreateBSSecret(secretName string, secretNamespace, npmRegistry string,
-	ctx context.Context, client client.Client) error {
+func HandleRHDHOperatorInstallation(ctx context.Context, client client.Client, olmClientSet olmclientset.Clientset, namespace string) error {
+	knativeLogger := log.FromContext(ctx)
```

Comment:
why `knativeLogger`? Should by `rhdhLogger` instead no?

Reply:
that's right typo, updated to `rhdhLogger`